### PR TITLE
[ADD] Create the 'Client Type' in the Project Event

### PR DESCRIPTION
**Pull request by jananjoy**

### CODE REVIEW

File: project_event/models/res_partner_category.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2018 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class PartnerCategory(models.Model):
+    _name = 'res.partner.category'
+    _inherit = ['res.partner.category']
```
Comment (eilst) [inline project_event/models/res_partner_category.py:8]: You don't need to specify the name if you are already inheriting and extending, only if you want to create a new table. It's ok, but you could remove this line

File: project_event/models/res_partner_category.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```
Comment (rimbendhaou) [inline project_event/models/res_partner_category.py]: No need in odoo11

File: project_event/models/res_partner_category_type.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
```
Comment (rimbendhaou) [inline project_event/models/res_partner_category_type.py]: no need in odoo 11

File: project_event/views/res_partner_category_type_view.xml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+    <record id="action_partner_category_type" model="ir.actions.act_window">
+        <field name="name">Res Partner Category Type</field>
+        <field name="res_model">res.partner.category.type</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <record id="category_type_form" model="ir.ui.view">
+        <field name="name">Contact Tags Form</field>
```
Comment (rimbendhaou) [inline project_event/views/res_partner_category_type_view.xml]: Client Type Form

File: project_event/views/res_partner_category_type_view.xml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+<odoo>
+
+    <record id="action_partner_category_type" model="ir.actions.act_window">
+        <field name="name">Res Partner Category Type</field>
+        <field name="res_model">res.partner.category.type</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <record id="category_type_form" model="ir.ui.view">
```
Comment (rimbendhaou) [inline project_event/views/res_partner_category_type_view.xml]: better to use this id id="res_partner_category_type_form"

File: project_event/views/res_partner_category_type_view.xml [hunk 6]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="action_partner_category_type" model="ir.actions.act_window">
+        <field name="name">Res Partner Category Type</field>
```
Comment (rimbendhaou) [inline project_event/views/res_partner_category_type_view.xml]: name: Client Type

File: project_event/views/res_partner_category_type_view.xml [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+        <field name="res_model">res.partner.category.type</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <record id="category_type_form" model="ir.ui.view">
+        <field name="name">Contact Tags Form</field>
+        <field name="model">res.partner.category.type</field>
+        <field name="arch" type="xml">
+            <form string="Category Type">
```
Comment (rimbendhaou) [inline project_event/views/res_partner_category_type_view.xml]: Client Type

File: project_event/views/res_partner_category_type_view.xml [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+                <sheet>
+                    <group>
+                        <field name="name"/>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="category_type_tree" model="ir.ui.view">
```
Comment (rimbendhaou) [inline project_event/views/res_partner_category_type_view.xml]: res_partner_category_type_tree

File: project_event/views/res_partner_category_type_view.xml [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+        <field name="name">Contact Tags Tree</field>
```
Comment (rimbendhaou) [inline project_event/views/res_partner_category_type_view.xml]: Client Types Tree

File: project_event/views/res_partner_category_type_view.xml [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+        <field name="model">res.partner.category.type</field>
+        <field name="arch" type="xml">
+            <tree string="">
```
Comment (rimbendhaou) [inline project_event/views/res_partner_category_type_view.xml]: string="Client Type"

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/models/res_partner_category_type.py [hunk 11]
```diff
@@ -0,0 +1,12 @@
+# © 2018 Savoir-faire Linux
```
Comment (rimbendhaou) [inline project_event/models/res_partner_category_type.py:1]: You have to add the access right for this new model

**Review by rimbendhaou — CHANGES_REQUESTED** [review]
need access right for the new model res.partner.category.type

File: project_event/models/res_partner_category_type.py (hunk 11 above)
Comment (jananjoy) [reply project_event/models/res_partner_category_type.py:1]: @rimbendhaou Done

**Comment by jananjoy** [comment]
I updated access right and change the string type too

**Review by rimbendhaou — APPROVED** [review]

**Pull request merged**